apps/web/pages/auth/login.tsx

```diff
@@ -94,6 +95,16 @@ export default function Login({
     </Button>
   );
 
+  const ExternalTotpFooter = (
+    <Button
+      onClick={() => {
+        window.location.replace("/");
```

Comment:
The only way we could reuse the two factor auth required flag is that when the user wants to cancel, we reload, **not** closing the modal as the "Go Back" button does when Cal credentials are used.